apps/frontend/src/views/Results.vue

```diff
@@ -468,6 +475,13 @@ export default class Results extends mixins(RouteMixin, ServerMixin) {
       this.evalInfo = file;
     }
   }
+
+  onUpdateGroups(newValue: string) {
+    this.updateGroups = newValue;
+    this.$nextTick(() => {
+      this.updateGroups = false;
+    })
+  }
```

Comment:
Groups being stored and updated in the vuex store would remove the need for this as well I believe.